### PR TITLE
Init ts setup

### PR DESCRIPTION
**Pull request by adrifdez**
First commit with a ts structure to start working on the project

### CODE REVIEW

File: src/views/app.style.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { createUseStyles } from "react-jss";
```
Comment (elias-garcia) [inline src/views/app.style.ts:1]: This file should be renamed to `app.styles.ts`

**Comment by elias-garcia** [comment]
When I run `npm install` the `package-lock.json` gets modified:

Captura de pantalla 2022-03-02 a las 11 48 05

Maybe @amonsosanz can confirm if this is happening to him as well.

**Comment by amonsosanz** [comment]
After running `npm i` I got a modified `package-lock.json`. What versions of node/npm are you using? Mine are:

```
$ node --version
v16.13.2

$ npm --version
8.1.2
```

**Comment by adrifdez** [comment]
> 

Could you try again? I've updated that file

**Review by elias-garcia — APPROVED** [review]

**Comment by amonsosanz** [comment]
> When I run `npm install` the `package-lock.json` gets modified:
> 
> Captura de pantalla 2022-03-02 a las 11 48 05
> 
> Maybe @amonsosanz can confirm if this is happening to him as well.

Same. What versions of node/npm are you using? Mine are:

```
$ node --version
v16.13.2

$ npm --version
8.1.2
```

**Review by amonsosanz — APPROVED** [review]

**Pull request merged**